allennlp_models/seq2seq/seq2seq_reader.py

```diff
@@ -109,23 +121,28 @@ def _read(self, file_path: str):
     def text_to_instance(
         self, source_string: str, target_string: str = None
     ) -> Instance:  # type: ignore
+        if self._ptb_sanitize_strings:
+            source_string = sanitize_ptb_tokenized_string(source_string)
+            target_string = sanitize_ptb_tokenized_string(target_string)
```

Comment:
Since you found a better source file, this is no longer necessary, right?

Reply:
To be sure, I want to keep the `sanitize_ptb_tokenized_string` function, but unless there is a compelling reason to add this functionality to the seq2seq reader, we don't have to make it in here.

Reply:
Yes, I wrote my own preprocessing code for the CNN/DM data, so I don't have to bother with the PTB tokenizer anymore. I removed those calls.